backend/dataall/core/environment/services/managed_iam_policies.py

```diff
@@ -42,6 +42,14 @@ def generate_empty_policy(self) -> dict:
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def create_managed_policy_from_inline_and_delete_inline(self) -> str:
+        """
+        Returns policy arn and needs to be implemented in the ManagedPolicies inherited classes
+        It is used for backwards compatibility. It should be deprecated and removed in future releases.
+        """
+        raise NotImplementedError
```

Comment:
nit: `raise NotImplementedError` isn't required, when there is at least one abstract method python will throw `TypeError: Can't instantiate abstract class T with abstract method ...` when you try to instantiate a derived class. So I suggest that since you are touching this file you might as well remove it from everywhere.

Reply:
True, I can leave the `...` !